src/rules/keyword_case.ts

```diff
@@ -59,15 +66,15 @@ export class KeywordsUpper extends ABAPRule {
 
     for (const statement of file.getStatements()) {
       if (statement.get() instanceof Unknown
-          || statement.get() instanceof MacroContent
-          || statement.get() instanceof MacroCall
-          || statement.get() instanceof Comment) {
+        || statement.get() instanceof MacroContent
```

Comment:
I actually like having the extra spaces here, so its easy for the eye to find the contents, will probably add a tslint/eslint rule sometime in the future

Reply:
It's an inconsistency between our formatters, do you happen to know where I can configure it to behave the same?

Reply:
I format these things manually 😄 but not a big problem for now